content/docs/querying/functions.md

```diff
@@ -178,6 +178,10 @@ rate(http_requests_total{job="api-server"}[5m])
 
 `rate` should only be used with counters.
 
+Note that when combining functions (e.g. `sum()`) with `rate()`, you must
```

Comment:
`sum` is actually not a function, but an aggregation operator.

Suggestion:

When combining `rate()` with an aggregation operator (e.g. `sum()`) or a function aggregating over time (any function that ends with `_over_time`), always take the `rate()` first and then aggregate, otherwise ...